operator/tests/common/central-cr.yaml

```diff
@@ -14,6 +14,15 @@ spec:
         cpu: 1
     adminPasswordSecret:
       name: admin-pass
+    db:
+      external: false
```

Comment:
It's not great user experience to require explicit specification of a false value. Can you please instead make this field optional and/or default to `false`?